### PR TITLE
feature/fixed_income

### PR DESCRIPTION
**Pull request by hemendra-sakpal-algobulls**

### CODE REVIEW

File: docs/member/fixed-income.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+To begin using the Fixed Income marketplace, download the AlgoBulls App or register on [www.algobulls.com](http://www.algobulls.com)
+
+After signing in, go to the Fixed Income section in the sidebar navigation. You will see the following fields:
+
+**i. Marketplace:** Find all the bonds, NCD & MLD cards in this marketplace. You can Buy or Sell these securities, view Security Details, use the Investment Calculator & view the Payment Cycle.
+
+[![Fixed Income](imgs/fixed_income_1.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/bonds_marketplace_1fixed_income_1.png)
+
+**ii. Fixed Income Card:** Get complete details, such as the Bond & NCD name, price, rating, maturity date, yield per year % and more here. Click on the know more to view the expanded version.
```
Comment (akhil-jain-algobulls) [inline docs/member/fixed-income.md:32]: `Click on the know more to view the expanded version.`

remove this

File: docs/member/fixed-income.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+[![Fixed Income](imgs/fixed_income_card.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/fixed_income_card.png)
+
+**iii. Search Bar:** To find a specific Security, enter its name or ISIN number into the search bar.
```
Comment (akhil-jain-algobulls) [inline docs/member/fixed-income.md:36]: `enter its name`

change it to

`enter the name`

File: docs/member/fixed-income.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+* **Portfolio:** This is your fixed income portfolio section where the security details, purchase date, Interest Payment Date, Total Cost Price, Total Current Value and Action are displayed.
+
+[![Fixed Income](imgs/fixed_income_portfolio.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/fixed_income_portfolio.png)
+
+**vii. Manage Accounts:** Manage your Fixed Income details such as add your Bank Account, Demat Details & KYC Information.
+
+## 4. How To Buy a security?
+
+It's crucial to update your KYC, bank information, and demat account information in the Manage Account section in the sidebar navigation before you Buy a security.
```
Comment (akhil-jain-algobulls) [inline docs/member/fixed-income.md:104]: `It's crucial to update your KYC`

change to

`It is crucial...`

Comment (akhil-jain-algobulls) [reply docs/member/fixed-income.md:104]: `Manage Account section in the sidebar navigation before you Buy a security.`

change to

`Manage Account section from the sidebar navigation before you buy a security.`

File: docs/member/fixed-income.md (hunk 1 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/fixed-income.md:32]: Done

File: docs/member/fixed-income.md (hunk 2 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/fixed-income.md:36]: Done

File: docs/member/fixed-income.md (hunk 3 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/fixed-income.md:104]: Done

File: docs/member/fixed-income.md [hunk 4]
```diff
@@ -101,7 +101,7 @@ Click on know more to get detailed information of the bond.
 
 ## 4. How To Buy a security?
 
-It's crucial to update your KYC, bank information, and demat account information in the Manage Account section in the sidebar navigation before you Buy a security.
+It is crucial to update your KYC, bank information, and demat account information in the Manage Account section in the sidebar navigation before you buy a security. in the sidebar navigation before you sell a security.
```
Comment (akhil-jain-algobulls) [inline docs/member/fixed-income.md:104]: `It is crucial to update your KYC, bank information, and demat account information in the Manage Account section in the sidebar navigation before you buy a security. in the sidebar navigation before you sell a security.`

please revisit this and change as per what you think is correct

**Review by akhil-jain-algobulls — APPROVED** [review]

**Pull request merged**